test/express-tests.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var path = require('path'),
+  helpers = require('yeoman-generator').test,
+  assert = require('yeoman-generator').assert,
+  temp = require('temp').track();
+
+describe('Express Sub Generators Tests', function () {
+  this.timeout(0);
```

Comment:
When I was initially PRing the changes, I had a problem with CI/CD integration - the tests failed.
As the main test already contained timeout disabling, I decided, that I would just go the same way and disable them too.